Integrations/integration-Threat_Grid.yml

```diff
@@ -1,7 +1,7 @@
 commonfields:
-  id: Threat Grid
+  id: Threat Grid New
```

Comment:
Why did you add `New` to `name` and `id` fields? As far as I understand this should keep backward compatibility.